assets/js/components/ClusterDetails/ClusterNodeLink.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { renderWithRouterMatch } from '@lib/test-utils';
+import ClusterNodeLink from '@components/ClusterDetails/ClusterNodeLink';
+
+describe('ClusterNodeLink', () => {
+  it('renders HostLink when hostId is provided', () => {
+    const hostId = '12345678-abcd-1234-abcd-1234567890ab';
+    const name = 'host01';
+    const path = '/hosts/:hostId';
+
+    renderWithRouterMatch(<ClusterNodeLink hostId={hostId} name={name} />, {
+      path,
```

Comment:
Do you need this `path` and `route` thing?
You are not clicking the url.
I think `renderWithRouter` would be enough